dax/controller/controller.go

```diff
@@ -784,6 +784,18 @@ func (c *Controller) Databases(ctx context.Context, orgID dax.OrganizationID, id
 	return c.Schemar.Databases(tx, orgID, ids...)
 }
 
+// get the list of workers being used by a database
+func (c *Controller) GetDatabaseWorkerCount(ctx context.Context, qdbid dax.QualifiedDatabaseID) (int, error) {
+
+	tx, err := c.Transactor.BeginTx(ctx, false)
+	if err != nil {
+		return -1, errors.Wrap(err, "beginning tx")
```

Comment:
this should return 0, not -1. Go convention is to return the zero value when there's a non-nil error

Reply:
(applies everywhere else as well)